src/binary-reader-objdump.cc

```diff
@@ -1781,11 +1809,23 @@ Result BinaryReaderObjdump::OnEventSymbol(Index index,
   if (name.empty()) {
     name = GetEventName(event_index);
   }
-  PrintDetails("   - [%d] E <" PRIstringview "> event=%" PRIindex, index,
+  PrintDetails("   - %d: E <" PRIstringview "> event=%" PRIindex, index,
```

Comment:
Looks like we don't have any test that tickles this?  (Since I don't see any corresponding test change due to this?)

Reply:
Indeed -- just noticed this as I had copied the table implementation from events, and events were printing inconsistently with regards to everything else, so I just fixed this one up.  Indeed, apparently no tests.